src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -107,86 +111,89 @@ const CreateConceptForm = props => (
               )}
             </div>
           </div>
-        )}
-      </div>
-      <div className="datatypefield">
-        <label htmlFor="datatype">Datatype</label>
-        <select
-          id="datatype"
-          name="datatype"
-          required
-          value={props.state.datatype}
-          className="form-control "
-          onChange={props.handleChange}
-        >
-          <option>Numeric</option>
-          <option>Text</option>
-          <option>None</option>
-          <option>Document</option>
-          <option>Date</option>
-          <option>Time</option>
-          <option>Datetime</option>
-          <option>Boolean</option>
-          <option>Rule</option>
-          <option>Structured-Numeric</option>
-          <option>Complex</option>
-          <option>Coded</option>
-        </select>
-      </div>
+          )}
+        </div>
+        <div className="datatypefield">
+          <label htmlFor="datatype">Datatype</label>
+          <select
+            id="datatype"
+            name="datatype"
+            required
+            value={props.state.datatype}
+            className="form-control "
+            onChange={props.handleChange}
+          >
+            <option>Numeric</option>
+            <option>Text</option>
+            <option>None</option>
+            <option>Document</option>
+            <option>Date</option>
+            <option>Time</option>
+            <option>Datetime</option>
+            <option>Boolean</option>
+            <option>Rule</option>
+            <option>Structured-Numeric</option>
+            <option>Complex</option>
+            <option>Coded</option>
+          </select>
+        </div>
```

Comment:
These too

Reply:
@Tittoh once again, it's the linter issue there.